cookiesCTF/main.py

```diff
@@ -0,0 +1,261 @@
+		<br></br>
+		<a href="/4" class="link">Task №4</a>
+		<br></br>
+		<a href="/5" class="link">Task №5</a>
+		<br></br>
+		<p></p>
+		<div class="form_design">
+			<p><b>Check flag:</b></p>
+			<form action="/" method="post" class="form">
+				<b>Flag:</b> <input name="flag" type="text" />
+				<br></br>
+				<input value="Check!" type="submit" />
+			</form>
+		</div>
+		<link rel="stylesheet" type="text/css" href="/styles.css">
+		<link rel="preconnect" href="https://fonts.gstatic.com">
+		<link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300&display=swap" rel="stylesheet">
+		<link href="https://fonts.googleapis.com/css2?family=Balsamiq+Sans:ital@1&display=swap" rel="stylesheet">
+	</body>
+    '''
+
+
+@get('/styles.css')
+def retCss():
+    return static_file('styles.css', root='./')
+
+
+@route('/', method='POST')
+def checkFlag():
+    flag = request.forms.get('flag')
+    if flag == "PB{eat_my_cookies}" or flag == "PB{I'm_a_cookiebot}" or flag == "PB{Jake_don't_mind}" or flag == "PB{Jake_isn't_lonely}" or flag == 'PB{Y0tub3_v1d30s_ar3_fun}':
+        return '''
+		<body>
+			<h1 class="correct">Correct!</h1>
+			<link rel="stylesheet" type="text/css" href="/styles.css">
+			<link rel="preconnect" href="https://fonts.gstatic.com">
+			<link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300&display=swap" rel="stylesheet">
+			<link href="https://fonts.googleapis.com/css2?family=Balsamiq+Sans:ital@1&display=swap" rel="stylesheet">
+		</body>'''
+    else:
+        return '''
+		<body>
+			<h1 class="try_again">Try again!</h1>
+			<link rel="stylesheet" type="text/css" href="/styles.css">
+			<link rel="preconnect" href="https://fonts.gstatic.com">
+			<link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300&display=swap" rel="stylesheet">
+			<link href="https://fonts.googleapis.com/css2?family=Balsamiq+Sans:ital@1&display=swap" rel="stylesheet">
+		</body>'''
+
+
+@route('/1')
+def task1():
+    response.set_cookie("is_admin", "no", path="/")
+    if request.get_cookie("is_admin") == "no":
+        return '''
+		<body>
+			<h1>Hello, Guest!</h1>
+			<link rel="stylesheet" type="text/css" href="/styles.css">
+			<link rel="preconnect" href="https://fonts.gstatic.com">
+			<link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300&display=swap" rel="stylesheet">
+			<link href="https://fonts.googleapis.com/css2?family=Balsamiq+Sans:ital@1&display=swap" rel="stylesheet">
+		</body>'''
+    elif request.get_cookie("is_admin") == "yes":
+        return '''
+		<body>
+			<h1>Hello, Admin!</h1>
+			<p>Flag: PB{eat_my_cookies}</p>
+			<link rel="stylesheet" type="text/css" href="/styles.css">
+			<link rel="preconnect" href="https://fonts.gstatic.com">
+			<link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300&display=swap" rel="stylesheet">
+			<link href="https://fonts.googleapis.com/css2?family=Balsamiq+Sans:ital@1&display=swap" rel="stylesheet">
+		</body>'''
```

Comment:
What's about `else`? You do not respond if the cookie is not what you expect? I think you should behave in the same way as if the cookie was "no"